src/server/scripts/EasternKingdoms/SunkenTemple/instance_sunken_temple.cpp

```diff
@@ -98,7 +96,6 @@ class instance_sunken_temple : public InstanceMapScript
                 case GO_ATALAI_STATUE4: GOAtalaiStatue4 = go->GetGUID();   break;
                 case GO_ATALAI_STATUE5: GOAtalaiStatue5 = go->GetGUID();   break;
                 case GO_ATALAI_STATUE6: GOAtalaiStatue6 = go->GetGUID();   break;
```

Comment:
According to TC codestyle, `switch` / `case` statements should have `break;` statements on their own line beneath the case line and with a 4 space indent. Not sure if it is preferred to leave it unchanged or make the improvement.